data/xapleaux/xapleaux weapons.txt

```diff
@@ -0,0 +1,249 @@
+	description "	An individual warhead for this weapon is very low yield, but the rate of fire is the key feature. A ship carrying several of these can flood the space between it and the target with an impenetrable wall of projectiles that are particularly effective at keeping swarms of small craft (or wildlife) at bay."
+
+outfit "Quininoid Active"
+	weapon
+		sprite "projectile/quininoid active"
+			"frame rate" 30
+		"hit effect" "missile hit"
+		"die effect" "missile death"
+		"velocity" 1
+		"lifetime" 100
+		"acceleration" 0.3
+		"turn" 5
+		"homing" 1
+		"radar tracking" 1
+		"split range" 200
+		"piercing" .2
+		"shield damage" 100
+		"hull damage" 125
+		"hit force" 400
+		"missile strength" 60
+		"submunition" "Quin" 5
+
+outfit "Quin"
+	weapon
+		sprite "projectile/quin"
+		"hit effect" "tiny explosion"
+		"die effect" "tiny explosion"
+		"velocity" 3
+		"lifetime" 60
+		"shield damage" 20
+		"hull damage" 25
+		"leak damage" .1
+		"hit force" 80
+		"inaccuracy" 25
+		"missile strength" 12
+
+
+outfit "Repulsor Wave Turret"
+	category "Turrets"
+	cost 1838000
+	thumbnail "outfit/repulsor wave turret"
+	"mass" 56
+	"outfit space" -56
+	"weapon capacity" -56
+	"required crew" 1
+	"turret mounts" -1
+	weapon
+		sprite "projectile/repulsor wave"
+			"frame rate" 8
+			"no repeat"
+		"hardpoint sprite" "hardpoint/repulsor wave turret"
+		"hardpoint offset" 9.
+		sound "repulsor wave"
+		"inaccuracy" 4
+		"turret turn" 7
+		"velocity" 10
+		"lifetime" 14
+		"reload" 15
+		"firing energy" 4
+		"firing heat" 1.4
+		"hit force" 10
+		"split range" 1
+		"submunition" "Repulsor Split" 10
+	description "This pulse wave turret imparts functionally no damage on a target but is very effective at keeping fauna native to The Brilliance from posing a hazard to civilian ships. As it does no damage it is not a military weapon at all, but does avoid angering swarms or otherwise breaking apart hazardous objects. As such it is frequently deployed by both military and civilians alike for general security purposes."
+
+outfit "Repulsor Split"
+	weapon
+		sprite "projectile/repulsor split"
+			"frame rate" 3
+			"no repeat"
+			#"scale" 0.8
+		"lifetime" 34
+		"hit force" 10
+		"inaccuracy" 15
+		"split range" 1
+		"submunition" "Repulsor Fade" 10
+		
+outfit "Repulsor Fade"
+	weapon
+		sprite "projectile/repulsor fade"
+			"frame rate" 2
+			"no repeat"
+			"scale" 0.8
+		"lifetime" 40
+		"hit force" 10
+		"inaccuracy" 10
+
+
+outfit "Xapleaux Anti-Missile"
+	category "Turrets"
+	cost 845000
+	thumbnail "outfit/warder"
+	"mass" 28
+	"outfit space" -28
+	"weapon capacity" -28
+	"energy capacity" 50
+	"turret mounts" -1
+	weapon
+		"hardpoint sprite" "hardpoint/warder"
+		"hardpoint offset" 3.
+		"hit effect" "xapleaux am"
+		"anti-missile" 10
+		"velocity" 400
+		"lifetime" 1
+		"reload" 8
+		"firing energy" 24
+		"firing heat" 10
+	description "This anti-missile system helps defend Xapleaux warships from long-range attacks."
```

Comment:
I assume this is just a placeholder, and wil get more details later.

Reply:
Nope that's all it is... at least until `plot` happens.